### DISCUSSION TITLE
Using scanner-document Capacitor in my Mobile Application

### DISCUSSION THREAD

**Discussion by allgonzalez** [quasarframework/quasar#13621] [CLI - Capacitor / Cordova modes]
Hi, I am developing an application and I need to add the option to scan documents, I am using "capacitor-scanner-documents" but when I add the plugin and run it tells me this error:"Not Implemented on Web" , I am working in Android Studio.
-At the moment I am just testing
![Codigo](https://user-images.githubusercontent.com/79773911/172212474-004d05c1-bfc7-4064-a149-ed54ab126ac5.png)
![error](https://user-images.githubusercontent.com/79773911/172213359-0a33eab1-375d-4319-9e89-93d4e9782cdf.png)

Plugin-Capacitor-Scanner-Document: https://www.npmjs.com/package/capacitor-document-scanner

Code in Quasar:

**Comment by MilosPaunovic** [comment]
This a not Quasar related, try raising an issue on the plugin repository: [https://github.com/WebsiteBeaver/capacitor-document-scanner/issues](https://github.com/WebsiteBeaver/capacitor-document-scanner/issues)
